runtime/caml/camlatomic.h

```diff
@@ -29,8 +29,9 @@
 extern "C++" {
 #include <atomic>
 #define ATOMIC_UINTNAT_INIT(x) (x)
-typedef std::atomic<uintnat> atomic_uintnat;
-typedef std::atomic<intnat> atomic_intnat;
+using atomic_uintnat = std::atomic<uintnat>;
+using atomic_intant = std::atomic<intnat>;
+using atomic_int64 = std::atomic<int64>;
```

Comment:
Is this valid C++?  

At any rate I don't think we need to define `atomic_int64` in a public header.  Just use `_Atomic int64` where needed.

Reply:
It's the "modern" typedef in C++ that also allows usage of template ([type_alias](https://en.cppreference.com/w/cpp/language/type_alias)).